src/views/Error.vue

```diff
@@ -22,15 +25,24 @@
       <!-- failsafe begins -->
       <div v-if="hasFailSafe">
 
+        <!-- TODO - Adding loading spinner till YT video loads -->
         <!-- failsafe G-form begins -->
-        <div v-if="isFailSafeGform">
+        <div v-if="isFailSafeGform && isVideoIdFetched" >
+          <hr class="solid">
+
           <div class="lead_text">
-            <p>इसी प्रतियोगिता में Google फॉर्म से हिस्सा लेने के लिए नीचे बटन पे क्लिक करें</p>
+            <p>इसी प्रतियोगिता में Google फॉर्म से हिस्सा लेने के लिए नीचे दिया गया वीडियो देखें और उसके बाद Google फॉर्म का लिंक क्लिक करें </p>
+            <div class='embed-container'>
+              <iframe :src="this.value['youtubeId']" id="lesson-video" frameborder='0' allowfullscreen></iframe>
+            </div>
+            <br>
+            <i class="far fa-hand-point-right"></i>
             <a
-              :href=value.failsafeUrl
-              class="icon-block">
-              <img src="../assets/google_form.svg">
```

Comment:
Removed the g-form link as we received feedback that it was non-intuitive for the students